ingress/vendor/k8s.io/kubernetes/pkg/client/unversioned/testclient/fake_componentstatuses.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
```

Comment:
This import where generated by `godep`. After the refactoring to vendor those file not referenced where removed. Without this update I see compilation errors running the tests